src/workspace-data/import-jobs.ts

```diff
@@ -27,29 +27,24 @@ export const useImportJobs = (workspace: WorkspaceWrapper): UseImportJobsResult
       workspace: { namespace, name },
     } = workspace;
     try {
-      const runningJobsInWorkspace: { jobId: string }[] = await Ajax(signal)
-        .Workspaces.workspace(namespace, name)
-        .listImportJobs(true);
-
-      asyncImportJobStore.update((previousState) => {
-        return [
-          ...previousState.filter((job) => !isJobInWorkspace(job, workspace)),
-          ...runningJobsInWorkspace.map(({ jobId }) => ({ jobId, targetWorkspace: { namespace, name } })),
-        ];
-      });
+      // Azure workspaces know about running jobs from the asyncImportJobStore
```

Comment:
I found this comment confusing... can you clarify how asyncImportJobStore finds out about the running jobs for Azure workspace?

Reply:
Also, the code in this block is also using `asyncImportJobStore`, so it's not clear how Azure workspaces are different.